### PR TITLE
Central tests - removed deploy on server

### PR DESCRIPTION
**Pull request by jkopriva**
Changes:
- removed deploy on server and all corresponding files
- added copyright + author where it was missing

### CODE REVIEW

**Comment by jkopriva** [comment]
@psrna @rawagner @rhopp please review:)

File: tests/org.jboss.tools.central.ui.bot.test/pom.xml [hunk 1]
```diff
@@ -14,13 +14,7 @@
...
 
 	<properties>
-		<jbosstools.test.wildfly.home>${requirementsDirectory}/wildfly-10.0.0.Final</jbosstools.test.wildfly.home>
-		<wildflySystemProperties></wildflySystemProperties>
-		<eapSystemProperties></eapSystemProperties>
-		<jbosstools.test.jboss-eap-6.4.home>${requirementsDirectory}/jboss-eap-6.4</jbosstools.test.jboss-eap-6.4.home>
-		<jbosstools.test.jboss-eap-7.0.home>${requirementsDirectory}/jboss-eap-7.0</jbosstools.test.jboss-eap-7.0.home>
-		<rd.config>${project.build.directory}/classes/server_properties</rd.config>
-		<systemProperties>${integrationTestsSystemProperties} ${wildflySystemProperties} ${eapSystemProperties} -Dusage_reporting_enabled=false -Drd.config=${rd.config} -Dmaven.config.file=${project.build.directory}/classes/settings.xml</systemProperties>
+		<systemProperties> -Dusage_reporting_enabled=false -Dmaven.config.file=${project.build.directory}/classes/settings.xml</systemProperties>
```
Comment (rawagner) [inline tests/org.jboss.tools.central.ui.bot.test/pom.xml]: usage reporting is already disabled in parent

File: tests/org.jboss.tools.central.ui.bot.test/pom.xml [hunk 2]
```diff
@@ -14,13 +14,7 @@
 	<packaging>eclipse-test-plugin</packaging>
 
 	<properties>
-		<jbosstools.test.wildfly.home>${requirementsDirectory}/wildfly-10.0.0.Final</jbosstools.test.wildfly.home>
-		<wildflySystemProperties></wildflySystemProperties>
-		<eapSystemProperties></eapSystemProperties>
-		<jbosstools.test.jboss-eap-6.4.home>${requirementsDirectory}/jboss-eap-6.4</jbosstools.test.jboss-eap-6.4.home>
-		<jbosstools.test.jboss-eap-7.0.home>${requirementsDirectory}/jboss-eap-7.0</jbosstools.test.jboss-eap-7.0.home>
-		<rd.config>${project.build.directory}/classes/server_properties</rd.config>
```
Comment (rawagner) [inline tests/org.jboss.tools.central.ui.bot.test/pom.xml]: so you dont use servers at all ? If yes, than you can also remove https://github.com/jbosstools/jbosstools-integration-tests/tree/master/tests/org.jboss.tools.central.ui.bot.test/resources/server_properties

Comment (rawagner) [inline tests/org.jboss.tools.central.ui.bot.test/pom.xml]: I see you already did that, nevermind :)

File: tests/org.jboss.tools.central.ui.bot.test/pom.xml (hunk 1 above)
Comment (rawagner) [inline tests/org.jboss.tools.central.ui.bot.test/pom.xml]: -Dmaven.config.file.. where is it used ? I see that you are using target/classes/settings.xml in your tests but not this property. So either you should use the property and change the test or you dont need this prop at all

**Comment by jkopriva** [comment]
@rawagner thx for review, all is fixed:)

**Pull request merged**